public/src/modules/api.js

```diff
@@ -55,7 +55,8 @@ define('api', () => {
 	api.post = (route, payload, onSuccess) => call({
 		url: route,
 		method: 'post',
-		data: payload,
+		data: JSON.stringify(payload),
+		contentType: 'application/json; charset=utf-8',
```

Comment:
Was it not possible to put this inside `call` if data is passed in?

Reply:
This seemed like the best way. Don't have to make conditional logic, don't have to worry about messing up get/head, it's more declarative, and it follows the repetition of the csrf.